### ISSUE TITLE
Uploading an image, inserting it into the post.

### ISSUE THREAD

**Issue by AliasScripts** [canbican/wordpress-java#74]
Hi!

First off thanks for this project, ridiculously useful.

Currently, I'm able to make posts with everything I need in them.. except an image.

Is it possible to:
1: Add an image into the post along with the rest of the text content
2: Make that image a link

Thanks for any help / tips!

**Comment by AliasScripts** [comment]
Hm I think I make have solved half of my own problem, enough to work with.

Since you can pass html as post content, if you know the image url you can construct a link:
`recentPost.setPost_content("<a href='http://google.com' target='_blank'><img src='http://i.imgur.com/DRmh6S9.jpg' alt=''></a>");`

Now I just need to figure out how to upload my images.. hmm..

**Comment by AliasScripts** [comment]
Found code to upload an image!

https://github.com/canbican/wordpress-java/issues/67#issuecomment-258135285

But I'm getting error 503? Strange considering I can make posts fine.

**Comment by AliasScripts** [comment]
Alright, so I have not been able to solve what causes the 503. I did, however, find a simple workaround using FTP and 1 more apache library. The code is short and easy too.

Library: http://commons.apache.org/proper/commons-net/download_net.cgi
Code: http://www.codejava.net/java-se/networking/ftp/java-ftp-file-upload-tutorial-and-example
(Check 3. Sample Code. It uploads two files with 2 methods. I just used the input stream method and it worked fine.)
